toolchain/lc-compile/src/check.g

```diff
@@ -1754,8 +1754,6 @@
         Id'Position -> Position
 		Id'Namespace -> Namespace
         (|
-			where(Namespace -> id(_))
-		||
```

Comment:
That would explain its presence...

So the aim is to warn people if they make a definition with name matching pattern _*[a-z]+. This means we only need to check definition id's (e.g. variable <id>) not all ids as definition ids are not allowed to be qualified.

Perhaps we just need to move the check :)